.github/workflows/validate-migration.yaml

```diff
@@ -79,28 +69,58 @@ jobs:
           
           noTool)
             # No tool is being used, so we don't need to set any flags
+            CMAKE_CXX_FLAGS="-stdlib=libc++"
           ;;
               
           addressAndUndefinedSanitizer)
-            OPENSSL_FLAGS="-fsanitize=address -fsanitize=undefined -fno-sanitize-recover=all -nostdinc++ -nostdlib++ -stdlib=libc++ -fPIE"
-            CMAKE_CXX_FLAGS="-fsanitize=address -fsanitize=undefined -fno-sanitize-recover=all"
+            BUILD_LIBCXX="true"
+            OPENSSL_FLAGS="-fsanitize=address -fsanitize=undefined -fno-sanitize-recover=all -nostdinc++ -nostdlib++ -stdlib=libc++ -fPIE -DPEDANTIC"
+            CMAKE_CXX_FLAGS="-fsanitize=address -fsanitize=undefined -fno-sanitize-recover=all -nostdinc++ -nostdlib++ -stdlib=libc++ -fPIE"
             CMAKE_EXE_LINKER_FLAGS="-lc++ -lc++abi -lunwind -pie"
             LLVM_USE_SANITIZER="Address;Undefined"
+          
+            # Use Clang's Sanitizer implementation
+            CC="sccache clang-16"
+            CXX="sccache clang++-16"
+            CMAKE_CXX_COMPILER="clang++-16"
+            CMAKE_C_COMPILER="clang-16"
           ;;
         
           threadSanitizer)
-            OPENSSL_FLAGS="-fsanitize=thread -fno-sanitize-recover=all -nostdinc++ -nostdlib++ -stdlib=libc++"
-            CMAKE_CXX_FLAGS="-fsanitize=thread -fno-sanitize-recover=all"
+            BUILD_LIBCXX="true"
+            OPENSSL_FLAGS="-fsanitize=thread -fno-sanitize-recover=all -nostdinc++ -nostdlib++ -fPIE -stdlib=libc++"
+            CMAKE_CXX_FLAGS="-fsanitize=thread -fno-sanitize-recover=all -nostdinc++ -nostdlib++ -fPIE -stdlib=libc++"
             CMAKE_EXE_LINKER_FLAGS="-lc++ -lc++abi -lunwind -pie"
             LLVM_USE_SANITIZER="Thread"
+          
+            # Use Clang's Sanitizer implementation
+            CC="sccache clang-16"
+            CXX="sccache clang++-16"
+            CMAKE_CXX_COMPILER="clang++-16"
+            CMAKE_C_COMPILER="clang-16"
```

Comment:
Below, in the `echo` lines being output to `GITHUB_ENV`, we're already doing an empty check, and if that's the case, then using `clang-16`. So we should be able to remove these lines, right?